docs/reference/buildx_create.md

```diff
@@ -140,6 +140,7 @@ Passes additional driver-specific options. Details for each driver:
   - `limits.cpu` - Sets the limit CPU value specified in units of Kubernetes CPU. Example `limits.cpu=100m`, `limits.cpu=2`
   - `limits.memory` - Sets the limit memory value specified in bytes or with a valid suffix. Example `limits.memory=500Mi`, `limits.memory=4G`
   - `nodeselector="label1=value1,label2=value2"` - Sets the kv of `Pod` nodeSelector. No Defaults. Example `nodeselector=kubernetes.io/arch=arm64`
+  - `tolerations="key=foo,value=bar;key2=foo2,operator=exists"` - Sets the `Pod` tolerations. Accepts the same values as the kube manifest tolerations. Key-value pairs are separated by `,`, tolerations are separated by `;`. No Defaults. Example `tolerations=operator=exists`
```

Comment:
Can we have `effect` examples?

Reply:
Added an example there. Would it make sense to add a link here to the relevant kube docs? https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/